workers.tf

```diff
@@ -103,7 +103,9 @@ resource "aws_autoscaling_group" "workers" {
     "capacity_rebalance",
     local.workers_group_defaults["capacity_rebalance"]
   )
-
+  timeouts {
+    delete = lookup(local.workers_group_defaults["timeouts"], "delete", null)
```

Comment:
this should be customized via `workers_group_defaults["timeouts"]` but also via `timeouts` setting per particulal worker definition as rest of parameters which are defined here https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/locals.tf

Reply:
added an extra lookup against the per particular worker definition

Reply:
This will work, but in general this is in other way as other variables. Please review and use same pattern which we arleady using